### ISSUE TITLE
Screen recording (GIF) doesn't produce any file to capture directory

### ISSUE THREAD

**Issue by simakuutio** [ShareX/ShareX#975]
Capture process itself seems to be going as expected but no files are created.

Debug log shows:

2015-09-06 12:37:37.175 - Starting Animated GIF recording. GIF encoding: "FFmpeg", FPS: 5
2015-09-06 12:37:41.949 - CLI: "C:\Users\drdoom\Documents\ShareX\Tools\ffmpeg.exe" -y -rtbufsize 100M -f gdigrab -framerate 5 -offset_x -785 -offset_y 354 -video_size 780x76 -draw_mouse 1 -i desktop -c:v libx264 -r 5 -preset ultrafast -tune zerolatency -qp 0 "C:\Users\drdoom\Documents\ShareX\Screenshots\2015-09\2015-09-06_12-37-41.mp4"
2015-09-06 12:37:55.365 - CLI: "C:\Users\drdoom\Documents\ShareX\Tools\ffmpeg.exe" -y -i "C:\Users\drdoom\Documents\ShareX\Screenshots\2015-09\2015-09-06_12-37-41.mp4" -vf "palettegen=stats_mode=full" "C:\Users\drdoom\Documents\ShareX\Tools\palette.png"

But directory remains empty.

**Comment by Jaex** [comment]
Is FFmpeg not shows any error? Can you try updating FFmpeg, using download button.

**Comment by simakuutio** [comment]
I did futher debugging. When capturing, that mp4 appears to capture directory and when I stop (not cancel) capture, .mp4 is deleted and directory remains empty. I'm expecting at least one .gif file to appear there but nothing comes as a result.

I admit, my original description of this bug isn't accurate enough, sorry about that. Anyway, expecting a .gif to be produced but that's not happening. ffmpeg doesn't give any errors.

**Comment by Jaex** [comment]
Did you tried updating FFmpeg?

**Comment by simakuutio** [comment]
No, not yet, just debugged what's happening under the hood. And no, didn't try to download an update because when ffmpeg is installed (I remember downloading it from sharex long time ago), it's no longer offering that option anywhere, I did check all menus several times. Of course, it must be under some option but I couldn't locate it immediately.

Update: I did rename old ffmpeg.exe to something else and sharex did offer an option to download it when tried to do screen recording. After update, capture was a success. So this is resolved. Thank you very much. This ticket can be closed.
